build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -525,6 +525,14 @@ export interface TabBarProps {
      * The default selected tab index
      */
     defaultTab?: number;
+    /**
+     * Whether to show or hide labels in the tab bar (Android Only)
+     * selected - only show on active tab
+     * auto - "show" when <4 tabs, "selected" when >3
+     * show - always show.
+     * hide - always hide (no labels)
+     */
+    labelVisibility?: "auto" | "show" | "hide" | "selected";
```

Comment:
Can you use 'labeled', 'unlabeled' and 'selected' instead to match the java constants?

Reply:
Yep can do - I was trying to make the props a little more 'user friendly' but happy to keep it lined up with the base constants.